src/logic/Utils.ts

```diff
@@ -63,34 +63,54 @@ export const parseIni = (data: string) => {
   };
   const value = {};
   const lines = data.split(/[\r\n]+/);
-  let section = null;
+  let section: string | null = null;
   lines.forEach(function(line) {
     if (regex.comment.test(line)) {
       return;
     } else if (regex.param.test(line)) {
-      const match = line.match(regex.param);
+      const match: string[] | null = line.match(regex.param);
 
       // TODO: github copilot made this part, but I have no idea what it does
       // if (match?.length !== 3) {
       //   throw "Could not parse INI file.";
       // }
 
       if (section) {
-        value[section][match[1]] = match[2];
+        value[section][match?.[1]] = match?.[2];
       } else {
-        value[match[1]] = match[2];
+        value[match![1]] = match?.[2];
```

Comment:
Any idea what's going on with the "Forbidden non-null assertion" stuff the lint action is throwing up around here?

Reply:
@theRealPadster No idea, but this leads to less errors than the alternative.